custom_components/tuya_local/devices/beok_tol47_thermostat.yaml

```diff
@@ -43,7 +43,7 @@ primary_entity:
           value: program
         - dps_val: wind
           value: temp_override
-          hidden: true
+          hidden: false

```

Comment:
Please keep changes to other configs separate.

I believe this was hidden because temporary override is not something you set by changing the mode, but is automatically set if you change another setting. So it is indicated here, but not available to select.